buildpatches/protoc_gen_protobufjs_use_toolchain.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/rules.bzl b/rules2.bzl
+index b4a6843..e610d99 100644
+--- a/rules.bzl
++++ b/rules2.bzl
+@@ -1,4 +1,5 @@
+ load("@rules_proto//proto:defs.bzl", "ProtoInfo")
++load("@rules_proto//proto:proto_common.bzl", proto_toolchains = "toolchains")
```

Comment:
The proto toolchain logic has migrated to `@protobuf//bazel/...`, so we could futureproof this by avoiding `rules_proto`. But since the redirects will remain in place, it's not urgent and we could merge this as is.

Reply:
I will do that as I upstream the change to the other repo, but for now, adding the logic here to remove rules_proto makes the patch messy.